webpack.config.js

```diff
@@ -0,0 +1,63 @@
+const defaultsDeep = require('lodash.defaultsdeep');
+const path = require('path');
+
+const base = {
+    mode: process.env.NODE_ENV === 'production' ? 'production' : 'development',
+    devtool: 'cheap-module-source-map',
+    entry: {
+        'scratch-render-fonts': './fonts.js'
+    },
+    module: {
+        rules: [{
+            include: path.resolve('src'),
+            test: /\.js$/,
+            loader: 'babel-loader',
+            options: {
+                presets: [['env', {targets: {}}]]
```

Comment:
Shouldn't there be targets here? Or, if it's just replaced by the "derived" configs, can you just omit this?

Reply:
I copied this almost exactly from https://github.com/LLK/scratch-2.0-coping-strategies/pull/15/files
I'm not really sure what it does